src/main/java/com/endava/soa_5ed/task2/regex/StringGenerator.java

```diff
@@ -0,0 +1,54 @@
+package com.endava.soa_5ed.task2.regex;
+
+public class StringGenerator {
+    /* Date Time and Regex - Exercise 1 */
+    final static String baseStr = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890,./;'[]\\<>?:\"{}|`~!@#$%^&*()_+-=";
+    int baseStrLen = baseStr.length();
+
+    public String getGenStr(int genStrLen) {
+        return generateString(genStrLen);
```

Comment:
Why not to put directly the functionality - without creating another method and call it.

Reply:
done.